### ISSUE TITLE
Removing registered trademark symbol

### ISSUE THREAD

**Issue by mattgarrish** [mattgarrish/epub-revision#1835]
IDPF legally required that the ® symbol appear after the first instance of "EPUB" in a specification that wasn't part of a specification name.

The symbol was removed from the overview document, so does that mean it's no longer a requirement and can be removed from the other specifications, or are we still required to keep it per IDPF practice and it should be restored?

**Comment by iherman** [comment]
I will follow this up when I'm back.

**Comment by iherman** [comment]
I am happy you could track down the precedence, @mattgarrish. I had a déjà vu feeling when you raised the question, but I did not remember, and I could not try things while on vacations. But it seems that it is indeed o.k. to remove the symbol. 

(Maybe, just for the records, it is worth putting here the mail reference of the earlier discussion and decision, in case it was public...)

**Comment by mattgarrish** [comment]
Agree, having a record for why we dropped IDPF conventions would be good. It took me a day to remember that we had already inquired about whether the IDPF trademark lines needed to be retained. Unfortunately, the discussion was posted to the chairs list, which I don't have access to. Even if it's not public, it would be good if someone could post the link so we at least know where it can be found.
